### DISCUSSION TITLE
Search Result item preview custom template

### DISCUSSION THREAD

**Discussion by jnhrv** [microsoft-search/pnp-modern-search#1496] [Q&A] [answered]
Hi,
I'm not able to find out the way how and if it is even possible to customize the result item's preview when clicking the PreviewURL from the simple list layout I started customizing.
I would like to remove some icons, like Download, Share, etc.

Is that even possible?

I'm a beginner with Modern Search and templating but could anyone navigate me the right direction or briefly instruct me on this?

Thank you! 

![chrome_snap_09-11-2021_13-58-34](https://user-images.githubusercontent.com/86712571/140928681-3786d2f8-0a7b-46f0-b6be-0836133a0b10.png)

**Comment by FranckyC** [comment, marked as answer]
Hi @hravic, the preview is not a page we own and is rendered within an iframe meaning you can't change the layout.

**Reply by jnhrv** [reply]
@FranckyC Thanks for the answer.
